src/pages/api/chatgpt/user.ts

```diff
@@ -33,6 +21,25 @@ type Response = {
   error?: string;
 };
 
+function encrypt(data: string, secret: string) {
+  const iv = Buffer.from('hgudlo8fbj$ng.fq');
+  const cipher = createCipheriv("aes-256-cbc", secret, iv);
+  let encrypted = cipher.update(data, "utf8", "hex");
+  encrypted += cipher.final("hex");
+  return iv.toString("hex") + ":" + encrypted;
+}
+
+export function decrypt(encrypted: string, secret: string) {
+  const [ivHex, encryptedHex] = encrypted.split(":");
+  const iv = Buffer.from(ivHex, "hex");
+  const decipher = createDecipheriv("aes-256-cbc", secret, iv);
+  let decrypted = decipher.update(encryptedHex, "hex", "utf8");
+  decrypted += decipher.final("utf8");
+  return decrypted;
+}
+
+export const secret = "BKc4rL.qwB8rzz7K@tXB9!uht3iKQWGV";
```

Comment:
?????

Reply:
just for testing, need to make it work first, will move another random one to vercel config